packages/odyssey-storybook/src/components/odyssey-labs/DataTable/DataTable.stories.tsx

```diff
@@ -526,7 +525,7 @@ export const NoActions: StoryObj<DataTableProps> = {
           hasSearch={props.hasSearch}
```

Comment:
I can't figure out how to add a comment there, but on (new) L516 I think you'll need to change `fetchDataFn` as you did on L406.